e2e/rbd.go

```diff
@@ -1984,6 +1986,111 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a encrypted block pvc and verify the image size", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					framework.Failf("failed to delete storageclass: %v", err)
+				}
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{"encrypted": "true", "encryptionType": util.EncryptionTypeBlock.String()},
+					deletePolicy)
+				if err != nil {
+					framework.Failf("failed to create storageclass: %v", err)
+				}
+
+				//nolint:goconst // The string "1Gi" is used multiple times in rbd.go, so it's not a const value.
+				var (
+					imageSize       uint64
+					resizeImageSize uint64
+				)
+				pvcSize := "1Gi"
+				if s, err := helpers.RoundUpToB(resource.MustParse(pvcSize)); err != nil {
```

Comment:
it should be `err==nil` not the other way isn't it?

Reply:
😀yea, missed it
Thanks!